doctors_surgery.html

```diff
@@ -84,51 +84,49 @@ <h2>Need advice or have some feedback for us?</h2>
 
         <p>Send us a message by filling out the form below and one of our Receptionists will be in touch:</p>
 
-        <form action="" method="get">
+        <form>
 
-            <div>
+            <fieldset>
                 <p>Are you completing this form on behalf of: </p>
 
                 <label for="self">Yourself </label>
-                <input type="checkbox" name="self">
+                <input type="checkbox" id="self" name="self">
 
                 <label for="another">Someone else </label>
-                <input type="checkbox" name="another">
-            </div>
+                <input type="checkbox" id="another" name="another">
+            </fieldset>
 
             <div>
```

Comment:
Part of me is wondering whether we need divs around each input and label - would they be used more for styling purposes?